test/common/run-tests

```diff
@@ -217,7 +219,7 @@ class GlobalMachine:
         # provide enough RAM for cryptsetup's PBKDF, as long as that is not configurable:
         # https://bugzilla.redhat.com/show_bug.cgi?id=1881829
         self.machine = machine_class(verbose=True, networking=self.networking, image=self.image, cpus=cpus,
-                                     memory_mb=memory_mb or 1400)
+                                     memory_mb=memory_mb or DEFAULT_MACHINE_MEMORY_SIZE)
```

Comment:
Sorry, I just realized that these constants are all messed up now, see https://github.com/cockpit-project/bots/pull/7946#issuecomment-3028618176

I recommend to just keep the 1400 here, or move that as a constant within run-tests as `NONDESTRUCTIVE_MACHINE_MEMORY_SIZE` (and then move the comment as well) -- but I think that's not actually necessary/helpful.